api_app/serializers.py

```diff
@@ -882,7 +882,15 @@ def get_attribute(self, instance: PluginConfig):
             if (
                 instance.is_secret()
                 and instance.for_organization
-                and self.context["request"].user.pk != instance.owner.pk
+                and not (
+                    self.context["request"].user.pk == instance.owner.pk
+                    or (
+                        self.context["request"].user.has_membership()
+                        and self.context["request"].user.membership.organization.pk
+                        == instance.owner.membership.organization.pk
+                        and self.context["request"].user.membership.is_admin
+                    )
+                )
```

Comment:
We need a comment here too, this is fucking me up.
```
We return `redacted` when
1) is a secret AND
2) is a value for the organization AND
3) we are not its owner  OR
4) we are not an admin of the same organization
```
if I did not fuck up the boolean expression, it seems good to me, just add a comment for `future us`